src/test/scala/com/advancedtelematic/campaigner/db/CampaignsSpec.scala

```diff
@@ -63,3 +65,50 @@ class CampaignsSpec extends AsyncFlatSpec
     } yield stats.finished shouldBe devices.length
   }
 }
+
+final class CampaignsFindFailedDevicesSpec extends AsyncFlatSpec
+  with DatabaseSpec
+  with Matchers
+  with ScalaFutures
+  with CampaignSupport
+  with UpdateSupport
+  with DatabaseUpdateSpecUtil
+  with CampaignerSpecUtil {
+
+  import Arbitrary._
+
+  val campaigns = Campaigns()
+  implicit val defaultPatience = PatienceConfig(timeout = Span(2, Seconds))
```

Comment:
If you want to get fancy: if you import `import org.scalatest.time.SpanSugar._` instead of `import org.scalatest.time.{Seconds, Span}` then you can write this as `PatienceConfig(timeout = 2 seconds)`.